.github/workflows/deploy_production_appstore.yml

```diff
@@ -51,6 +51,13 @@ jobs:
         touch KMMIC/Configurations/Plists/GoogleService/Production/GoogleService-Info.plist
         echo $IOS_GOOGLE_SERVICE_INFO_PRODUCTION | base64 --decode > ./KMMIC/Configurations/Plists/GoogleService/Production/GoogleService-Info.plist
 
+    - name: Setup Konfig Properties
+      env:
+       BUILD_KONFIG_PROPERTIES: ${{ secrets.BUILD_KONFIG_PROPERTIES }}
+      run: |
+       touch ../buildKonfig.properties
+       echo $BUILD_KONFIG_PROPERTIES | base64 --decode > ../buildKonfig.properties
```

Comment:
Did you add `buildKonfig.properties` to gitignore?

Reply:
@phongvhd93 I already did it [here](https://github.com/markgravity/kmm-ic/blob/99945a33e341dbb801ff394748f06686a1d05da9/.gitignore#L139)